sdcm/nemesis.py

```diff
@@ -5246,6 +5246,38 @@ def disrupt_grow_shrink_zero_nodes(self):
         znode = random.choice([node for node in self.cluster.zero_nodes if node.dc_idx == self.target_node.dc_idx])
         self.decommission_nodes(nodes=[znode])
 
+    @target_all_nodes
+    def disrupt_serial_restart_elected_topology_coordinator(self):
+        """ Serial restart of elected topology coordinator node,
+        should trigger new coordinator node election
+        """
+        if not self.target_node.raft.is_consistent_topology_changes_enabled:
+            raise UnsupportedNemesis("Consistent topology changes feature is disabled")
+
+        self.use_nemesis_seed()
+        num_of_restarts = random.randint(1, len(self.cluster.nodes))
+        self.log.debug("Number of serial restart of topology coordinator: %s", num_of_restarts)
+        election_wait_timeout = random.choice([1, 5, 10, 15])
```

Comment:
There are several randoms here without a "seed".
If one case detects an issue, it would be very hard to reproduce it without controlling the seed.

Reply:
we probably should refactor `use_nemesis_seed` to return a random object, and not assume it's setting the seed globally 

@soyacz one more item for refactoring list :)